docs/.vitepress/vitepress/components/globals/contributors.vue

```diff
@@ -0,0 +1,46 @@
+<script setup lang="ts">
+import { useData } from 'vitepress'
+import { computed } from 'vue'
+// @ts-expect-error missing types
+import _contributors from '/virtual-contributors'
+
+const props = defineProps<{ id: string }>()
+const { theme } = useData()
+
+const contributors = computed(() => _contributors[props.id])
+</script>
+
+<template>
+  <div class="mb-4">
+    <div class="flex flex-wrap gap-4 pt-2">
+      <div v-for="c of contributors" :key="c.hash">
+        <a
+          :href="`https://github.com/${theme.repo}/commits/${
+            theme.branch
+          }/packages/components/${id}?author=${encodeURIComponent(c.email)}`"
+          target="_blank"
+          class="flex gap-2 items-center link"
+        >
+          <img
+            :src="`https://gravatar.com/avatar/${c.hash}?d=retro`"
+            class="w-8 h-8 rounded-full"
+          />
+          {{ c.name }}
+        </a>
+      </div>
```

Comment:
Should we use the component created before in `.vitepress/vitepress/components/common/vp-link.vue`?